src/status_im/chat/commands/core.cljs

```diff
@@ -0,0 +1,139 @@
+
+(defview send-preview
+  [{:keys [content timestamp-str outgoing group-chat]}]
+  (letsubs [network [:network-name]]
+    (let [{{:keys [amount fiat-amount tx-hash asset] send-network :network} :params} content
+          recipient-name (get-in content [:params :bot-db :public :recipient])
+          network-mismatch? (and (seq send-network) (not= network send-network))]
+      [react/view style/command-send-message-view
+       [react/view
+        [react/view style/command-send-amount-row
+         [react/view style/command-send-amount
+          [react/text {:style style/command-send-amount-text
+                       :font  :medium}
+           amount
+           [react/text {:style (style/command-amount-currency-separator outgoing)}
+            "."]
+           [react/text {:style (style/command-send-currency-text outgoing)
+                        :font  :default}
+            asset]]]]
+        (when fiat-amount
+          [react/view style/command-send-fiat-amount
+           [react/text {:style style/command-send-fiat-amount-text}
+            (str "~ " fiat-amount " " (i18n/label :usd-currency))]])
+        (when (and group-chat
+                   recipient-name)
+          [react/text {:style style/command-send-recipient-text}
+           (str
+            (i18n/label :send-sending-to)
+            " "
+            recipient-name)])
+        [react/view
+         [react/text {:style (style/command-send-timestamp outgoing)}
+          (str (i18n/label :sent-at) " " timestamp-str)]]
+        [send-status tx-hash outgoing]
+        (when network-mismatch?
+          [react/text send-network])]])))
+
+(deftype PersonalSendCommand []
+  Command
+  (id [_]
+    :send)
+  (scope [_]
+    #{:personal-chats})
+  (parameters [_]
+    [{:id :asset
+      :type :text
+      ;; Suggestion components should be structured in such way that they will just take
+      ;; one argument, event-creator fn used to construct event to fire whenever something
+      ;; is selected.
+      ;; `:set-command-argument` will be event event defined in the chat input namespace,
+      ;; taking parameter id and selected value as argument and correctly setting them in
+      ;; the `[:chats chat-id :staged-command :parameters parameter-id]` db path.
+      ;;
+      ;; Example:
+      #_:suggestions #_(suggestions/choose-asset #(vector :set-command-argument :asset %))}
+     {:id :amount
+      :type :number}])
+  (validate [_ _ _]
+    ;; There is no validation for the `/send` command, as it's fully delegated to the wallet
+    nil)
+  (on-send [_ message-id parameters cofx]
+    ;; TODO janherich: express effect prefilling wallet `:wallet-send` state and navigating
+    ;; to wallet
+    (when-let [tx-hash (get-in cofx [:db :wallet :send-transaction :tx-hash])]
+      {:dispatch [:update-transactions]}))
+  (short-preview [_ command-message]
+    (send-short-preview command-message))
+  (preview [_ command-message]
+    (send-preview command-message)))
```

Comment:
On second thought, if messages are really just functions with react markup, they're probably decoupled enough.

Reply:
I totally agree, `preview` and `view` instead of the `short-preview` and `preview` would be much more fitting actually.

Reply:
That would've made sense if had we done it originally. At this point swapping preview to mean something different would be confusing probably. Maybe just `short-view` and `view`. Maps nicely to the current naming, just drop the `pre`